Build failure: dublin-traceroute

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

```
nix-shell -p dublin-traceroute
```

### Can Hydra reproduce this build failure?

Please select the Hydra Status.

### Link to Hydra build job

_No response_

### Relevant log output

```console
these 2 derivations will be built:
  /nix/store/l7av27jksxn45jzfbjvsgp418qfv9zp4-libtins-4.5.drv
  /nix/store/d9i5xnpbzm303ai9v3am8jr3fb9ypd04-dublin-traceroute-0.4.2-unstable-2024-04-11.drv
these 3 paths will be fetched (0.51 MiB download, 1.07 MiB unpacked):
  /nix/store/dn5qxhggil5ria58dizky0xz5rc6wv4s-jsoncpp-1.9.6
  /nix/store/5fplf90vrbfdgzlv98832jiz1ii44kv9-jsoncpp-1.9.6-dev
  /nix/store/xqbjl8n1nqcx9sdkzpr7s1vywyjcwpbd-source
copying path '/nix/store/xqbjl8n1nqcx9sdkzpr7s1vywyjcwpbd-source' from 'https://cache.nixos.org'...
copying path '/nix/store/dn5qxhggil5ria58dizky0xz5rc6wv4s-jsoncpp-1.9.6' from 'https://cache.nixos.org'...
waiting for lock on '/nix/store/z2482a9pbz7zxn752ramyswbmngkmyng-libtins-4.5'...
copying path '/nix/store/5fplf90vrbfdgzlv98832jiz1ii44kv9-jsoncpp-1.9.6-dev' from 'https://cache.nixos.org'...
building '/nix/store/l7av27jksxn45jzfbjvsgp418qfv9zp4-libtins-4.5.drv'...
Running phase: unpackPhase
unpacking source archive /nix/store/qzmm1ccs73rf75as700x1pmadm4fb9qr-source
source root is source
Running phase: patchPhase
applying patch /nix/store/hhvzjg99jzba9d0g6hdb09ss489dj1gw-0001-force-cpp-14.patch
patching file CMakeLists.txt
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_LOCALEDIR=/nix/store/z2482a9pbz7zxn752ramyswbmngkmyng-libtins-4.5/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/z2482a9pbz7zxn752ramyswbmngkmyng-libtins-4.5/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/z2482a9pbz7zxn752ramyswbmngkmyng-libtins-4.5/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/z2482a9pbz7zxn752ramyswbmngkmyng-libtins-4.5/share/doc/libtins -DCMAKE_INSTALL_INFODIR=/nix/store/z2482a9pbz7zxn752ramyswbmngkmyng-libtins-4.5/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/z2482a9pbz7zxn752ramyswbmngkmyng-libtins-4.5/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/z2482a9pbz7zxn752ramyswbmngkmyng-libtins-4.5/include -DCMAKE_INSTALL_SBINDIR=/nix/store/z2482a9pbz7zxn752ramyswbmngkmyng-libtins-4.5/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/z2482a9pbz7zxn752ramyswbmngkmyng-libtins-4.5/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/z2482a9pbz7zxn752ramyswbmngkmyng-libtins-4.5/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/7nlf5v84s4p2yhx327j8495yik60qnzh-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/7nlf5v84s4p2yhx327j8495yik60qnzh-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/7nlf5v84s4p2yhx327j8495yik60qnzh-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/z2482a9pbz7zxn752ramyswbmngkmyng-libtins-4.5
CMake Deprecation Warning at CMakeLists.txt:1 (CMAKE_MINIMUM_REQUIRED):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.


-- The C compiler identification is GNU 14.3.0
-- The CXX compiler identification is GNU 14.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/7nlf5v84s4p2yhx327j8495yik60qnzh-gcc-wrapper-14.3.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/7nlf5v84s4p2yhx327j8495yik60qnzh-gcc-wrapper-14.3.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Using specified 'Release' build type.
-- Build will generate a shared library. Use LIBTINS_BUILD_SHARED=0 to perform a static build
-- Found PCAP: /nix/store/bdxjlah82wy9mmqa4mvd20dpnya1lpa2-libpcap-1.10.5/lib/libpcap.so
-- Performing Test PCAP_LINKS_SOLO
-- Performing Test PCAP_LINKS_SOLO - Success
-- Looking for pcap_get_pfring_id
-- Looking for pcap_get_pfring_id - not found
-- Looking for pcap_set_immediate_mode
-- Looking for pcap_set_immediate_mode - found
-- Looking for pcap_set_tstamp_precision
-- Looking for pcap_set_tstamp_precision - found
CMake Deprecation Warning at cmake/Modules/CheckCXXFeatures.cmake:53 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.
Call Stack (most recent call first):
  CMakeLists.txt:97 (INCLUDE)


-- Performing Test _HAS_CXX11_FLAG
-- Performing Test _HAS_CXX11_FLAG - Success
-- Checking C++ support for "decltype"
-- Checking C++ support for "decltype": works
-- Checking C++ support for "declval"
-- Checking C++ support for "declval": works
-- Checking C++ support for "initializer_list"
-- Checking C++ support for "initializer_list": works
-- Checking C++ support for "rvalue-references"
-- Checking C++ support for "rvalue-references": works
-- Checking C++ support for "functional"
-- Checking C++ support for "functional": works
-- Checking C++ support for "chrono"
-- Checking C++ support for "chrono": works
-- Checking C++ support for "noexcept"
-- Checking C++ support for "noexcept": works
-- Checking C++ support for "builtin-swap"
-- Checking C++ support for "builtin-swap": works
-- Using C++11 features
-- Enabling IEEE 802.11 support.
-- Found OpenSSL: /nix/store/wadq4gzybbvg5243d90l9x4mw3459r3a-openssl-3.5.1/lib/libcrypto.so (found version "3.5.1")
-- Enabling WPA2 decryption support.
-- Enabling TCPIP classes
CMake Warning (dev) at CMakeLists.txt:150 (FIND_PACKAGE):
  Policy CMP0167 is not set: The FindBoost module is removed.  Run "cmake
  --help-policy CMP0167" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found Boost: /nix/store/ir0758kmac2mk5852s7cp65ss8ins92p-boost-1.87.0-dev/lib/cmake/Boost-1.87.0/BoostConfig.cmake (found version "1.87.0")
-- Enabling TCP ACK tracking support.
-- Enabling TCP stream custom data support.
CMake Warning (dev) at examples/CMakeLists.txt:2 (FIND_PACKAGE):
  Policy CMP0167 is not set: The FindBoost module is removed.  Run "cmake
  --help-policy CMP0167" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found Boost: /nix/store/ir0758kmac2mk5852s7cp65ss8ins92p-boost-1.87.0-dev/lib/cmake/Boost-1.87.0/BoostConfig.cmake (found version "1.87.0") found components: regex
-- Tests have been enabled
-- Found Threads: TRUE
-- Configuring done (5.1s)
-- Generating done (0.2s)
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_INSTALL_DOCDIR
    CMAKE_INSTALL_INCLUDEDIR
    CMAKE_INSTALL_INFODIR
    CMAKE_INSTALL_LIBEXECDIR
    CMAKE_INSTALL_LOCALEDIR
    CMAKE_INSTALL_MANDIR
    CMAKE_INSTALL_SBINDIR


-- Build files have been written to: /build/source/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
build flags: -j4 SHELL=/nix/store/p79bgyzmmmddi554ckwzbqlavbkw07zh-bash-5.2p37/bin/bash
[  0%] Creating directories for 'googletest'
[  0%] No download step for 'googletest'
[  2%] No update step for 'googletest'
[  2%] No patch step for 'googletest'
[  5%] Performing configure step for 'googletest'
loading initial cache file /build/source/build/googletest-prefix/tmp/googletest-cache-Release.cmake
CMake Warning (dev) at /build/source/build/googletest-prefix/tmp/googletest-cache-Release.cmake:2 (set):
  implicitly converting 'bool' to 'STRING' type.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at /build/source/build/googletest-prefix/tmp/googletest-cache-Release.cmake:3 (set):
  implicitly converting 'bool' to 'STRING' type.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at /build/source/build/googletest-prefix/tmp/googletest-cache-Release.cmake:4 (set):
  implicitly converting 'bool' to 'STRING' type.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at /build/source/build/googletest-prefix/tmp/googletest-cache-Release.cmake:5 (set):
  implicitly converting 'path' to 'STRING' type.
This warning is for project developers.  Use -Wno-dev to suppress it.

-- The C compiler identification is GNU 14.3.0
-- The CXX compiler identification is GNU 14.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/7nlf5v84s4p2yhx327j8495yik60qnzh-gcc-wrapper-14.3.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/7nlf5v84s4p2yhx327j8495yik60qnzh-gcc-wrapper-14.3.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Configuring done (0.8s)
-- Generating done (0.0s)
-- Build files have been written to: /build/source/build/googletest
[  5%] Performing build step for 'googletest'
[ 25%] Building CXX object googletest/CMakeFiles/gtest.dir/src/gtest-all.cc.o
[ 50%] Linking CXX static library ../lib/libgtest.a
[ 50%] Built target gtest
[ 75%] Building CXX object googletest/CMakeFiles/gtest_main.dir/src/gtest_main.cc.o
[100%] Linking CXX static library ../lib/libgtest_main.a
[100%] Built target gtest_main
[  8%] No install step for 'googletest'
[ 11%] Completed 'googletest'
[ 11%] Built target googletest
[ 13%] Building CXX object src/CMakeFiles/tins.dir/bootp.cpp.o
[ 16%] Building CXX object src/CMakeFiles/tins.dir/crypto.cpp.o
[ 16%] Building CXX object src/CMakeFiles/tins.dir/address_range.cpp.o
[ 13%] Building CXX object src/CMakeFiles/tins.dir/arp.cpp.o
[ 16%] Building CXX object src/CMakeFiles/tins.dir/detail/address_helpers.cpp.o
[ 19%] Building CXX object src/CMakeFiles/tins.dir/detail/icmp_extension_helpers.cpp.o
[ 19%] Building CXX object src/CMakeFiles/tins.dir/detail/pdu_helpers.cpp.o
/build/source/src/crypto.cpp: In member function 'Tins::SNAP* Tins::Crypto::WPA2::SessionKeys::ccmp_decrypt_unicast(const Tins::Dot11Data&, Tins::RawPDU&) const':
/build/source/src/crypto.cpp:501:24: warning: 'int AES_set_encrypt_key(const unsigned char*, int, AES_KEY*)' is deprecated: Since OpenSSL 3.0 [-Wdeprecated-declarations]
  501 |     AES_set_encrypt_key(&ptk_[0] + 32, 128, &ctx);
      |     ~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /build/source/src/crypto.cpp:38:
/nix/store/jyi9ix8mpg6zb22d14ibsxblyz7qm9lw-openssl-3.5.1-dev/include/openssl/aes.h:51:5: note: declared here
   51 | int AES_set_encrypt_key(const unsigned char *userKey, const int bits,
      |     ^~~~~~~~~~~~~~~~~~~
/build/source/src/crypto.cpp:519:16: warning: 'void AES_encrypt(const unsigned char*, unsigned char*, const AES_KEY*)' is deprecated: Since OpenSSL 3.0 [-Wdeprecated-declarations]
  519 |     AES_encrypt(counter, MIC, &ctx);
      |     ~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~
/nix/store/jyi9ix8mpg6zb22d14ibsxblyz7qm9lw-openssl-3.5.1-dev/include/openssl/aes.h:57:6: note: declared here
   57 | void AES_encrypt(const unsigned char *in, unsigned char *out,
      |      ^~~~~~~~~~~
/build/source/src/crypto.cpp:521:16: warning: 'void AES_encrypt(const unsigned char*, unsigned char*, const AES_KEY*)' is deprecated: Since OpenSSL 3.0 [-Wdeprecated-declarations]
  521 |     AES_encrypt(MIC, MIC, &ctx);
      |     ~~~~~~~~~~~^~~~~~~~~~~~~~~~
/nix/store/jyi9ix8mpg6zb22d14ibsxblyz7qm9lw-openssl-3.5.1-dev/include/openssl/aes.h:57:6: note: declared here
   57 | void AES_encrypt(const unsigned char *in, unsigned char *out,
      |      ^~~~~~~~~~~
/build/source/src/crypto.cpp:523:16: warning: 'void AES_encrypt(const unsigned char*, unsigned char*, const AES_KEY*)' is deprecated: Since OpenSSL 3.0 [-Wdeprecated-declarations]
  523 |     AES_encrypt(MIC, MIC, &ctx);
      |     ~~~~~~~~~~~^~~~~~~~~~~~~~~~
/nix/store/jyi9ix8mpg6zb22d14ibsxblyz7qm9lw-openssl-3.5.1-dev/include/openssl/aes.h:57:6: note: declared here
   57 | void AES_encrypt(const unsigned char *in, unsigned char *out,
      |      ^~~~~~~~~~~
/build/source/src/crypto.cpp:527:16: warning: 'void AES_encrypt(const unsigned char*, unsigned char*, const AES_KEY*)' is deprecated: Since OpenSSL 3.0 [-Wdeprecated-declarations]
  527 |     AES_encrypt(counter, crypted_block, &ctx);
      |     ~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/jyi9ix8mpg6zb22d14ibsxblyz7qm9lw-openssl-3.5.1-dev/include/openssl/aes.h:57:6: note: declared here
   57 | void AES_encrypt(const unsigned char *in, unsigned char *out,
      |      ^~~~~~~~~~~
/build/source/src/crypto.cpp:538:20: warning: 'void AES_encrypt(const unsigned char*, unsigned char*, const AES_KEY*)' is deprecated: Since OpenSSL 3.0 [-Wdeprecated-declarations]
  538 |         AES_encrypt(counter, crypted_block, &ctx );
      |         ~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/jyi9ix8mpg6zb22d14ibsxblyz7qm9lw-openssl-3.5.1-dev/include/openssl/aes.h:57:6: note: declared here
   57 | void AES_encrypt(const unsigned char *in, unsigned char *out,
      |      ^~~~~~~~~~~
/build/source/src/crypto.cpp:543:20: warning: 'void AES_encrypt(const unsigned char*, unsigned char*, const AES_KEY*)' is deprecated: Since OpenSSL 3.0 [-Wdeprecated-declarations]
  543 |         AES_encrypt(MIC, MIC, &ctx);
      |         ~~~~~~~~~~~^~~~~~~~~~~~~~~~
/nix/store/jyi9ix8mpg6zb22d14ibsxblyz7qm9lw-openssl-3.5.1-dev/include/openssl/aes.h:57:6: note: declared here
   57 | void AES_encrypt(const unsigned char *in, unsigned char *out,
      |      ^~~~~~~~~~~
[ 22%] Building CXX object src/CMakeFiles/tins.dir/detail/sequence_number_helpers.cpp.o
[ 22%] Building CXX object src/CMakeFiles/tins.dir/dhcp.cpp.o
[ 22%] Building CXX object src/CMakeFiles/tins.dir/dhcpv6.cpp.o
[ 25%] Building CXX object src/CMakeFiles/tins.dir/dns.cpp.o
[ 25%] Building CXX object src/CMakeFiles/tins.dir/dot3.cpp.o
[ 27%] Building CXX object src/CMakeFiles/tins.dir/dot1q.cpp.o
[ 27%] Building CXX object src/CMakeFiles/tins.dir/eapol.cpp.o
[ 30%] Building CXX object src/CMakeFiles/tins.dir/ethernetII.cpp.o
[ 30%] Building CXX object src/CMakeFiles/tins.dir/handshake_capturer.cpp.o
[ 30%] Building CXX object src/CMakeFiles/tins.dir/hw_address.cpp.o
[ 33%] Building CXX object src/CMakeFiles/tins.dir/icmp_extension.cpp.o
[ 33%] Building CXX object src/CMakeFiles/tins.dir/icmp.cpp.o
[ 36%] Building CXX object src/CMakeFiles/tins.dir/icmpv6.cpp.o
[ 36%] Building CXX object src/CMakeFiles/tins.dir/ip_reassembler.cpp.o
[ 38%] Building CXX object src/CMakeFiles/tins.dir/ip.cpp.o
[ 38%] Building CXX object src/CMakeFiles/tins.dir/ip_address.cpp.o
[ 38%] Building CXX object src/CMakeFiles/tins.dir/ipv6.cpp.o
[ 41%] Building CXX object src/CMakeFiles/tins.dir/ipv6_address.cpp.o
[ 41%] Building CXX object src/CMakeFiles/tins.dir/ipsec.cpp.o
[ 44%] Building CXX object src/CMakeFiles/tins.dir/llc.cpp.o
[ 44%] Building CXX object src/CMakeFiles/tins.dir/loopback.cpp.o
[ 47%] Building CXX object src/CMakeFiles/tins.dir/mpls.cpp.o
[ 47%] Building CXX object src/CMakeFiles/tins.dir/memory_helpers.cpp.o
[ 47%] Building CXX object src/CMakeFiles/tins.dir/network_interface.cpp.o
[ 50%] Building CXX object src/CMakeFiles/tins.dir/packet_sender.cpp.o
[ 50%] Building CXX object src/CMakeFiles/tins.dir/pdu.cpp.o
[ 52%] Building CXX object src/CMakeFiles/tins.dir/pdu_iterator.cpp.o
[ 52%] Building CXX object src/CMakeFiles/tins.dir/pdu_option.cpp.o
[ 52%] Building CXX object src/CMakeFiles/tins.dir/pppoe.cpp.o
[ 55%] Building CXX object src/CMakeFiles/tins.dir/radiotap.cpp.o
[ 55%] Building CXX object src/CMakeFiles/tins.dir/rawpdu.cpp.o
[ 58%] Building CXX object src/CMakeFiles/tins.dir/rsn_information.cpp.o
[ 58%] Building CXX object src/CMakeFiles/tins.dir/sll.cpp.o
[ 61%] Building CXX object src/CMakeFiles/tins.dir/snap.cpp.o
[ 61%] Building CXX object src/CMakeFiles/tins.dir/stp.cpp.o
[ 61%] Building CXX object src/CMakeFiles/tins.dir/tcp.cpp.o
[ 63%] Building CXX object src/CMakeFiles/tins.dir/tcp_ip/ack_tracker.cpp.o
[ 63%] Building CXX object src/CMakeFiles/tins.dir/tcp_ip/flow.cpp.o
[ 66%] Building CXX object src/CMakeFiles/tins.dir/tcp_ip/data_tracker.cpp.o
[ 66%] Building CXX object src/CMakeFiles/tins.dir/tcp_ip/stream.cpp.o
[ 69%] Building CXX object src/CMakeFiles/tins.dir/tcp_ip/stream_follower.cpp.o
[ 69%] Building CXX object src/CMakeFiles/tins.dir/tcp_ip/stream_identifier.cpp.o
[ 69%] Building CXX object src/CMakeFiles/tins.dir/timestamp.cpp.o
[ 72%] Building CXX object src/CMakeFiles/tins.dir/udp.cpp.o
[ 72%] Building CXX object src/CMakeFiles/tins.dir/utils/checksum_utils.cpp.o
[ 75%] Building CXX object src/CMakeFiles/tins.dir/utils/frequency_utils.cpp.o
[ 75%] Building CXX object src/CMakeFiles/tins.dir/utils/radiotap_parser.cpp.o
[ 77%] Building CXX object src/CMakeFiles/tins.dir/utils/radiotap_writer.cpp.o
[ 77%] Building CXX object src/CMakeFiles/tins.dir/utils/routing_utils.cpp.o
[ 77%] Building CXX object src/CMakeFiles/tins.dir/utils/resolve_utils.cpp.o
[ 80%] Building CXX object src/CMakeFiles/tins.dir/utils/pdu_utils.cpp.o
[ 80%] Building CXX object src/CMakeFiles/tins.dir/vxlan.cpp.o
[ 83%] Building CXX object src/CMakeFiles/tins.dir/dot11/dot11_base.cpp.o
[ 83%] Building CXX object src/CMakeFiles/tins.dir/dot11/dot11_data.cpp.o
[ 83%] Building CXX object src/CMakeFiles/tins.dir/dot11/dot11_mgmt.cpp.o
[ 86%] Building CXX object src/CMakeFiles/tins.dir/dot11/dot11_beacon.cpp.o
[ 86%] Building CXX object src/CMakeFiles/tins.dir/dot11/dot11_assoc.cpp.o
[ 88%] Building CXX object src/CMakeFiles/tins.dir/dot11/dot11_auth.cpp.o
[ 88%] Building CXX object src/CMakeFiles/tins.dir/dot11/dot11_probe.cpp.o
[ 91%] Building CXX object src/CMakeFiles/tins.dir/sniffer.cpp.o
[ 91%] Building CXX object src/CMakeFiles/tins.dir/dot11/dot11_control.cpp.o
[ 91%] Building CXX object src/CMakeFiles/tins.dir/packet_writer.cpp.o
[ 94%] Building CXX object src/CMakeFiles/tins.dir/pktap.cpp.o
[ 94%] Building CXX object src/CMakeFiles/tins.dir/tcp_stream.cpp.o
[ 97%] Building CXX object src/CMakeFiles/tins.dir/offline_packet_filter.cpp.o
[ 97%] Building CXX object src/CMakeFiles/tins.dir/ppi.cpp.o
[100%] Linking CXX shared library ../nix/store/z2482a9pbz7zxn752ramyswbmngkmyng-libtins-4.5/lib/libtins.so
[100%] Built target tins
buildPhase completed in 40 seconds
Running phase: checkPhase
check flags: SHELL=/nix/store/p79bgyzmmmddi554ckwzbqlavbkw07zh-bash-5.2p37/bin/bash VERBOSE=y tests test
/nix/store/hvn9cprr3p61gjlahk9pq4bmp3zaxj4a-cmake-3.31.7/bin/cmake -S/build/source -B/build/source/build --check-build-system CMakeFiles/Makefile.cmake 0
make  -f CMakeFiles/Makefile2 tests
make[1]: Entering directory '/build/source/build'
/nix/store/hvn9cprr3p61gjlahk9pq4bmp3zaxj4a-cmake-3.31.7/bin/cmake -S/build/source -B/build/source/build --check-build-system CMakeFiles/Makefile.cmake 0
/nix/store/hvn9cprr3p61gjlahk9pq4bmp3zaxj4a-cmake-3.31.7/bin/cmake -E cmake_progress_start /build/source/build/CMakeFiles 86
make  -f CMakeFiles/Makefile2 tests/src/CMakeFiles/tests.dir/all
make[2]: Entering directory '/build/source/build'
make  -f CMakeFiles/googletest.dir/build.make CMakeFiles/googletest.dir/depend
make[3]: Entering directory '/build/source/build'
cd /build/source/build && /nix/store/hvn9cprr3p61gjlahk9pq4bmp3zaxj4a-cmake-3.31.7/bin/cmake -E cmake_depends "Unix Makefiles" /build/source /build/source /build/source/build /build/source/build /build/source/build/CMakeFiles/googletest.dir/DependInfo.cmake "--color="
make[3]: Leaving directory '/build/source/build'
make  -f CMakeFiles/googletest.dir/build.make CMakeFiles/googletest.dir/build
make[3]: Entering directory '/build/source/build'
make[3]: Nothing to be done for 'CMakeFiles/googletest.dir/build'.
make[3]: Leaving directory '/build/source/build'
[  4%] Built target googletest
make  -f src/CMakeFiles/tins.dir/build.make src/CMakeFiles/tins.dir/depend
make[3]: Entering directory '/build/source/build'
cd /build/source/build && /nix/store/hvn9cprr3p61gjlahk9pq4bmp3zaxj4a-cmake-3.31.7/bin/cmake -E cmake_depends "Unix Makefiles" /build/source /build/source/src /build/source/build /build/source/build/src /build/source/build/src/CMakeFiles/tins.dir/DependInfo.cmake "--color="
Dependencies file "src/CMakeFiles/tins.dir/address_range.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/arp.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/bootp.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/crypto.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/detail/address_helpers.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/detail/icmp_extension_helpers.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/detail/pdu_helpers.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/detail/sequence_number_helpers.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/dhcp.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/dhcpv6.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/dns.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/dot11/dot11_assoc.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/dot11/dot11_auth.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/dot11/dot11_base.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/dot11/dot11_beacon.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/dot11/dot11_control.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/dot11/dot11_data.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/dot11/dot11_mgmt.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/dot11/dot11_probe.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/dot1q.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/dot3.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/eapol.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/ethernetII.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/handshake_capturer.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/hw_address.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/icmp.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/icmp_extension.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/icmpv6.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/ip.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/ip_address.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/ip_reassembler.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/ipsec.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/ipv6.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/ipv6_address.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/llc.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/loopback.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/memory_helpers.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/mpls.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/network_interface.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/offline_packet_filter.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/packet_sender.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/packet_writer.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/pdu.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/pdu_iterator.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/pdu_option.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/pktap.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/ppi.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/pppoe.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/radiotap.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/rawpdu.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/rsn_information.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/sll.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/snap.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/sniffer.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/stp.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/tcp.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/tcp_ip/ack_tracker.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/tcp_ip/data_tracker.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/tcp_ip/flow.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/tcp_ip/stream.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/tcp_ip/stream_follower.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/tcp_ip/stream_identifier.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/tcp_stream.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/timestamp.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/udp.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/utils/checksum_utils.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/utils/frequency_utils.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/utils/pdu_utils.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/utils/radiotap_parser.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/utils/radiotap_writer.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/utils/resolve_utils.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/utils/routing_utils.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/vxlan.cpp.o.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/tins.dir/link.d" is newer than depends file "/build/source/build/src/CMakeFiles/tins.dir/compiler_depend.internal".
Consolidate compiler generated dependencies of target tins
make[3]: Leaving directory '/build/source/build'
make  -f src/CMakeFiles/tins.dir/build.make src/CMakeFiles/tins.dir/build
make[3]: Entering directory '/build/source/build'
make[3]: Nothing to be done for 'src/CMakeFiles/tins.dir/build'.
make[3]: Leaving directory '/build/source/build'
[ 41%] Built target tins
make  -f tests/src/CMakeFiles/wpa2_decrypt_test.dir/build.make tests/src/CMakeFiles/wpa2_decrypt_test.dir/depend
make[3]: Entering directory '/build/source/build'
cd /build/source/build && /nix/store/hvn9cprr3p61gjlahk9pq4bmp3zaxj4a-cmake-3.31.7/bin/cmake -E cmake_depends "Unix Makefiles" /build/source /build/source/tests/src /build/source/build /build/source/build/tests/src /build/source/build/tests/src/CMakeFiles/wpa2_decrypt_test.dir/DependInfo.cmake "--color="
make[3]: Leaving directory '/build/source/build'
make  -f tests/src/CMakeFiles/wpa2_decrypt_test.dir/build.make tests/src/CMakeFiles/wpa2_decrypt_test.dir/build
make[3]: Entering directory '/build/source/build'
[ 41%] Building CXX object tests/src/CMakeFiles/wpa2_decrypt_test.dir/wpa2_decrypt_test.cpp.o
cd /build/source/build/tests/src && /nix/store/7nlf5v84s4p2yhx327j8495yik60qnzh-gcc-wrapper-14.3.0/bin/g++  -I/build/source/include -I/build/source/tests/src/../include -I/build/source/googletest/googletest/include -Wall -Wextra -std=c++14 -O3 -DNDEBUG -MD -MT tests/src/CMakeFiles/wpa2_decrypt_test.dir/wpa2_decrypt_test.cpp.o -MF CMakeFiles/wpa2_decrypt_test.dir/wpa2_decrypt_test.cpp.o.d -o CMakeFiles/wpa2_decrypt_test.dir/wpa2_decrypt_test.cpp.o -c /build/source/tests/src/wpa2_decrypt_test.cpp
In file included from /build/source/googletest/googletest/include/gtest/gtest-message.h:57,
                 from /build/source/googletest/googletest/include/gtest/gtest-assertion-result.h:46,
                 from /build/source/googletest/googletest/include/gtest/gtest.h:63,
                 from /build/source/tests/src/wpa2_decrypt_test.cpp:5:
/build/source/googletest/googletest/include/gtest/internal/gtest-port.h:273:2: error: #error C++ versions less than C++17 are not supported.
  273 | #error C++ versions less than C++17 are not supported.
      |  ^~~~~
In file included from /build/source/googletest/googletest/include/gtest/gtest.h:67:
/build/source/googletest/googletest/include/gtest/gtest-param-test.h:483:56: error: missing template arguments before '(' token
  483 |           typename StdFunction = decltype(std::function(std::declval<Func>()))>
      |                                                        ^
/build/source/googletest/googletest/include/gtest/gtest-param-test.h:493:56: error: missing template arguments before '(' token
  493 |           typename StdFunction = decltype(std::function(std::declval<Func>()))>
      |                                                        ^
/build/source/googletest/googletest/include/gtest/gtest-param-test.h: In function 'auto testing::ConvertGenerator(Gen&&, Func&&)':
/build/source/googletest/googletest/include/gtest/gtest-param-test.h:497:6: warning: 'if constexpr' only available with '-std=c++17' or '-std=gnu++17' [-Wc++17-extensions]
  497 |   if constexpr (is_single_arg_std_function) {
      |      ^~~~~~~~~
make[3]: *** [tests/src/CMakeFiles/wpa2_decrypt_test.dir/build.make:79: tests/src/CMakeFiles/wpa2_decrypt_test.dir/wpa2_decrypt_test.cpp.o] Error 1
make[3]: Leaving directory '/build/source/build'
make[2]: *** [CMakeFiles/Makefile2:2996: tests/src/CMakeFiles/wpa2_decrypt_test.dir/all] Error 2
make[2]: Leaving directory '/build/source/build'
make[1]: *** [CMakeFiles/Makefile2:991: tests/src/CMakeFiles/tests.dir/rule] Error 2
make[1]: Leaving directory '/build/source/build'
make: *** [Makefile:439: tests] Error 2
error: builder for '/nix/store/l7av27jksxn45jzfbjvsgp418qfv9zp4-libtins-4.5.drv' failed with exit code 2;
       last 25 log lines:
       > In file included from /build/source/googletest/googletest/include/gtest/gtest-message.h:57,
       >                  from /build/source/googletest/googletest/include/gtest/gtest-assertion-result.h:46,
       >                  from /build/source/googletest/googletest/include/gtest/gtest.h:63,
       >                  from /build/source/tests/src/wpa2_decrypt_test.cpp:5:
       > /build/source/googletest/googletest/include/gtest/internal/gtest-port.h:273:2: error: #error C++ versions less than C++17 are not supported.
       >   273 | #error C++ versions less than C++17 are not supported.
       >       |  ^~~~~
       > In file included from /build/source/googletest/googletest/include/gtest/gtest.h:67:
       > /build/source/googletest/googletest/include/gtest/gtest-param-test.h:483:56: error: missing template arguments before '(' token
       >   483 |           typename StdFunction = decltype(std::function(std::declval<Func>()))>
       >       |                                                        ^
       > /build/source/googletest/googletest/include/gtest/gtest-param-test.h:493:56: error: missing template arguments before '(' token
       >   493 |           typename StdFunction = decltype(std::function(std::declval<Func>()))>
       >       |                                                        ^
       > /build/source/googletest/googletest/include/gtest/gtest-param-test.h: In function 'auto testing::ConvertGenerator(Gen&&, Func&&)':
       > /build/source/googletest/googletest/include/gtest/gtest-param-test.h:497:6: warning: 'if constexpr' only available with '-std=c++17' or '-std=gnu++17' [-Wc++17-extensions]
       >   497 |   if constexpr (is_single_arg_std_function) {
       >       |      ^~~~~~~~~
       > make[3]: *** [tests/src/CMakeFiles/wpa2_decrypt_test.dir/build.make:79: tests/src/CMakeFiles/wpa2_decrypt_test.dir/wpa2_decrypt_test.cpp.o] Error 1
       > make[3]: Leaving directory '/build/source/build'
       > make[2]: *** [CMakeFiles/Makefile2:2996: tests/src/CMakeFiles/wpa2_decrypt_test.dir/all] Error 2
       > make[2]: Leaving directory '/build/source/build'
       > make[1]: *** [CMakeFiles/Makefile2:991: tests/src/CMakeFiles/tests.dir/rule] Error 2
       > make[1]: Leaving directory '/build/source/build'
       > make: *** [Makefile:439: tests] Error 2
       For full logs, run:
         nix log /nix/store/l7av27jksxn45jzfbjvsgp418qfv9zp4-libtins-4.5.drv
error: 1 dependencies of derivation '/nix/store/d9i5xnpbzm303ai9v3am8jr3fb9ypd04-dublin-traceroute-0.4.2-unstable-2024-04-11.drv' failed to build
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.37, NixOS, 25.11 (Xantusia), 25.11pre-git`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.4`
 - channels(root): `"nixos, nixpkgs"`
 - nixpkgs: `/nix/store/m305283ray2vnd2n01q228lb718kykh2-nixos/nixos`

### Notify maintainers

@baloo 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/du/dublin-traceroute/package.nix
+++ b/pkgs/by-name/du/dublin-traceroute/package.nix
@@ -23,6 +23,12 @@ stdenv.mkDerivation {
     hash = "sha256-UJeFPVi3423Jh72fVk8QbLX1tTNAQ504xYs9HwVCkZc=";
   };
 
+  # gtest requires C++17, while dublin-traceroute requires C++11
+  postPatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace-fail "ENABLE_TESTING()" ""
+  '';
+
   nativeBuildInputs = [
     cmake
     pkg-config

